python/tests/unit/test_cache.py

```diff
@@ -465,6 +465,54 @@ def test_long_cluster_1():
 
 MadnessVerifier = Callable[[Tuple[IR, EnvoyConfig]], bool]
 
+def test_mappings_same_name_delta():
+    # Tests that multiple Mappings with the same name (but in different namespaces)
+    # are properly added/removed from the cache when they are updated.
+    builder = Builder(logger, "cache_test_4.yaml")
+    b = builder.build()
+    econf = b[1]
+    econf = econf.as_dict()
+
+    # loop through all the clusters in the resulting envoy config and pick out two Mappings from our test set (first and lase)
+    # to ensure their clusters were generated properly.
+    cluster1_ok, cluster2_ok = False
+    for cluster in econf['static_resources']['clusters']:
+        cname = cluster.get('per_connection_buffer_limit_bytes', None)
```

Comment:
Why `per_connection_buffer_limit_bytes`?

Reply:
Whoops, good catch. That should just be the `name`. I was hasty in copying a section I wrote in the buffer limit bytes test that I happened to know looped over the envoy config. The fact that this did not fail revealed more disturbingly that none of these cache tests actually run :woman_facepalming:.